uw-frame-components/portal/features/controllers.js

```diff
@@ -53,10 +53,10 @@ define(['angular','require'], function(angular, require) {
 
 
      //local functions ---------------------------------------------------------
-     
+
      var getPopups = function(){
        portalFeaturesService.getUnseenPopups().then(function(unSeenPopups) {
-         if(unSeenPopups.length !=0){
+          if(unSeenPopups.length !=0 && !$rootScope.GuestMode===true){
```

Comment:
Can `!$rootScope.GuestMode===true` be idiomatically refactored to `!$rootScope.GuestMode`?